languages-and-frameworks/dockerfile.html.markerb

```diff
@@ -72,39 +79,51 @@ The values aren't displayed since they are secret!
 
 ## _Deploy Your App_
 
-If you didn't previously deploy the app, you can do that now.
+If you didn't previously deploy the app, or you've made changes and want to redeploy, then you can do that now.
 
 ```cmd
 fly deploy
 ```
 ```output
-Validating /Users/someone/project/fly.toml
+==> Verifying app config
+Validating /Users/name/test/my-app/fly.toml
 Platform: machines
 ✓ Configuration is valid
+--> Verified app config
 ==> Building image
-Remote builder fly-builder-restless-shadow-3063 ready
-==> Creating build context
---> Creating build context done
+Remote builder fly-builder-long-glitter-7257 ready
 ==> Building image with Docker
---> docker host: 20.10.12 linux x86_64
-Sending build context to Docker daemon     346B
+
 ...
-  Finished deploying
 
-Visit your newly deployed app at https://wispy-breeze-3322.fly.dev/
+--> Pushing image done
+image: registry.fly.io/my-app:deployment-01HGEQ1Z1DPMJ4NMPNHTA09T6T
+image size: 141 MB
+
+Watch your deployment at https://fly.io/apps/my-app/monitoring
+
+-------
+Updating existing machines in 'my-app' with rolling strategy
+
+-------
+ ✔ [1/2] Machine 918543b477de83 [app] update succeeded
+ ✔ [2/2] Machine e28697ce6d3986 [app] update succeeded
+-------
+
+Visit your newly deployed app at https://my-app.fly.dev/
 ```
 
-By default, `fly deploy` builds the image using a remote builder. If you have Docker running locally and want to build locally, then run `fly deploy --local-only`. Once your container is built, it's deployed!
+By default, `fly deploy` builds the image using a remote builder. If you have Docker running locally and want to build locally, then run `fly deploy --local-only`. After the image is built, the app gets deployed.
 
 ## _Open Your App_
 
-Run `fly apps open` to open your deployed app in a browser.
+Run `fly apps open`, or just go to the URL specified in the output, to open your deployed app in a browser.
 
 You're off and running!
 
 ## _Taking it Further_
 
 Lots of applications deployed in a container have some state that they want to keep. Here are a couple resources to check out for ways to do that.
```

Comment:
```suggestion
Lots of apps have some state that they want to keep. Here are a couple resources to check out for ways to do that.
```
meh?

Reply:
yeah I was trying to get rid of "container" without making things wrong. This is good!